ui/page/components/components.go

```diff
@@ -111,82 +111,93 @@ func UniformHorizontalPadding(gtx layout.Context, body layout.Widget) layout.Dim
+		txStatus.Icon = l.Theme.Icons.ReceiveIcon
+	default:
+		txStatus.Title = values.String(values.StrTransferred)
+		txStatus.Icon = l.Theme.Icons.Transferred
+	}
+
+	// replace icon for staking tx types
+	if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterStaking) {
+		switch tx.Type {
+		case dcrlibwallet.TxTypeTicketPurchase:
+			{
+				if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterUnmined) {
+					txStatus.Title = values.String(values.StrUmined)
+					txStatus.Icon = l.Theme.Icons.TicketUnminedIcon
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusUnmined
+					txStatus.Color = l.Theme.Color.LightBlue6
+					txStatus.Background = l.Theme.Color.LightBlue
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterImmature) {
+					txStatus.Title = values.String(values.StrImmature)
+					txStatus.Icon = l.Theme.Icons.TicketImmatureIcon
+					txStatus.Color = l.Theme.Color.LightBlue6
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusImmature
+					txStatus.ProgressBarColor = l.Theme.Color.LightBlue5
+					txStatus.ProgressTrackColor = l.Theme.Color.LightBlue3
+					txStatus.Background = l.Theme.Color.LightBlue
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterLive) {
+					txStatus.Title = values.String(values.StrLive)
+					txStatus.Icon = l.Theme.Icons.TicketLiveIcon
+					txStatus.Color = l.Theme.Color.Primary
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusLive
+					txStatus.ProgressBarColor = l.Theme.Color.Primary
+					txStatus.ProgressTrackColor = l.Theme.Color.LightBlue4
+					txStatus.Background = l.Theme.Color.Primary50
+				} else if wal.TxMatchesFilter(tx, dcrlibwallet.TxFilterExpired) {
+					txStatus.Title = values.String(values.StrExpired)
+					txStatus.Icon = l.Theme.Icons.TicketExpiredIcon
+					txStatus.Color = l.Theme.Color.GrayText2
+					txStatus.TicketStatus = dcrlibwallet.TicketStatusExpired
+					txStatus.Background = l.Theme.Color.Gray4
 				} else {
-					txStatus.Title = values.String(values.StrRevoked)
-					txStatus.Icon = l.Theme.Icons.TicketRevokedIcon
-					txStatus.Color = l.Theme.Color.Orange
-					txStatus.TicketStatus = dcrlibwallet.TicketStatusVotedOrRevoked
-					txStatus.ProgressBarColor = l.Theme.Color.Danger
-					txStatus.ProgressTrackColor = l.Theme.Color.Orange3
-					txStatus.Background = l.Theme.Color.Orange2
+					if ticketSpender != nil {
+						if ticketSpender.Type == dcrlibwallet.TxTypeVote {
+							txStatus.Title = values.String(values.StrTicketVotedTitle)
+							txStatus.Icon = l.Theme.Icons.TicketLiveIcon
+							txStatus.Color = l.Theme.Color.Turquoise700
+							txStatus.TicketStatus = dcrlibwallet.TicketStatusVotedOrRevoked
+							txStatus.ProgressBarColor = l.Theme.Color.Turquoise300
+							txStatus.ProgressTrackColor = l.Theme.Color.Turquoise100
+							txStatus.Background = l.Theme.Color.Success2
+						} else {
+							txStatus.Title = values.String(values.StrTicketRevokedTitle)
+							txStatus.Icon = l.Theme.Icons.TicketLiveIcon
```

Comment:
Live and revoked tickets, have similar icons... is this intentional?
<img width="401" alt="image" src="https://user-images.githubusercontent.com/27733432/175473388-e754b13b-0b18-41d1-b47a-a400e0de7441.png">

Reply:
this is seen on the staking page.

Reply:
![Screenshot from 2022-06-24 11-18-20](https://user-images.githubusercontent.com/25265396/175515504-b6961b1e-4e9d-498c-b9df-06822cccc1e2.png)

Decrediton uses same icon for tickets whether revoked or voted, so i did same.

Personally i feel we need to have the designers run through the tickets live cycle and provide is unique icons for all, plus i don't think the same method that populate the transactions list, should also populate the ticket list. there are overlaps.

Initially i was making the change of separating the methods in this PR, but later determined it would be best suited in another PR and properly detailed

Reply:
So if i understand correctly, the current icon for revoked tickets does not apply here then.

I feel regardless, we should maintain out icons for revoked as it was pending when we get designs for this by the design team. Except you have some concerns about it so we look into it objectively.

About the different methods for transaction and tickets, could you help expand on that so we know what we are up against and to help determine the best approach. it will also help us properly prioritize the recommended task.

Reply:
No the icon is only reflected on a revocation transaction. but all tickets have same icon

Right now, there's no standard icon in place, i mean if we want to we can revert it, but I think this works for now, i could probably work on this aspect for the v2 UI
![Screenshot from 2022-06-29 20-39-26](https://user-images.githubusercontent.com/25265396/176522910-c89a669a-095f-4ccc-8ea4-7b30a4fdd03c.png)


we should have 2 methods `renderTransactions` and `renderTicket`, naming is just for example purposes,  also a different method populating the ticket/transaction details page. just to give an example off the top of my head, the color change on the tickets list was intended for the transaction list only. also using the same methods doesn't allow for the transaction list, the ticket list and their details page to have different titles